### PR TITLE
RS-206 - Elasticsearch connector init

### PR DESCRIPTION
**Pull request by lagache**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,52 +1,130 @@
...
-## END
 
-# reshuffle-_CONNECTOR_NAME_-connector
+### Reshuffle Elasticsearch Connector
 
-### Reshuffle _CONNECTOR_NAME_ Connector
+This package contains a Reshuffle connector for connecting to [Elasticsearch](https://www.elastic.co/elasticsearch/).
 
-This connector provides <description>.
+The following example add a new entry to an existing index:
```
Comment (foundingnimo) [inline README.md]: adds

File: README.md [hunk 2]
```diff
@@ -1,52 +1,130 @@
...
 
-##### event1 description
-The connector fires this event when ...
+#### Connector Actions
 
-##### event2 description
-The connector fires this event when ...
+[SDK](#sdk) - Retrieve a Node JS Elasticsearch client
+
+[Examples using the SDK](#sdk)
```
Comment (foundingnimo) [inline README.md]: The link should point to `(#examples)` :)

File: README.md [hunk 3]
```diff
@@ -1,52 +1,130 @@
...
+#### <a name="setup"></a>Setup Cloud Elasticsearch
+Setup Elastic cloud:
+
+1. Go to https://cloud.elastic.co/home
+2. Click `Create deployment`
+3. Provide a name and create
+4. Copy the username/password   
+4. Click on your deployment in the deployment list
+5. copy the cloud ID which is <deployment-name>:<key>
+6. See [Configuration Options](#configuration) below on how to connect to this Elasticsearch cloud instance.
```
Comment (foundingnimo) [inline README.md]: I think 'for how' rather than 'on how' ?

File: README.md [hunk 4]
```diff
@@ -1,52 +1,130 @@
...
+
+#### <a name="configuration"></a>Configuration Options
+
+The Elasticsearch connector takes the same options as the Node JS Elasticsearch client.
+See [Node JS Elasticsearch Client options](https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/client-configuration.html)
+
+As an example, for connecting to elasticsearch cloud, provide options as below:
+
+```typescript
+import ElasticsearchConnector from "./index";
```
Comment (foundingnimo) [inline README.md]: Do we normally add the import statement in the Readme? I think we shouldn't

File: README.md [hunk 5]
```diff
@@ -1,52 +1,130 @@
...
+  },
+})
+```
 
 #### Connector actions
 
-##### action1
-The connector provides action1 which ...
+All actions are provided via the sdk.
+// See full list of actions documentations in [Node JS Elasticsearch Client API reference](https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/api-reference.html#api-reference)
```
Comment (foundingnimo) [inline README.md]: I suggest 'See full list of actions' or 'full list of actions documentation'

File: package.json [hunk 6]
```diff
@@ -13,7 +13,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/reshufflehq/reshuffle-_CONNECTOR_NAME_-connector.git"
+    "url": "git+https://github.com/reshufflehq/reshuffle-elasticsearch-connector.git"
   },
   "keywords": [
     "Reshuffle",
```
Comment (foundingnimo) [inline package.json:19]: Maybe add ElasticSearch as a keyword?

**Review by foundingnimo — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (lagache) [reply README.md]: lol no, good catch, Intellij tried to be smart again

**Review by foundingnimo — APPROVED** [review]

**Pull request merged**